recipe/thirdparty/api/providersForTenant.go

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package api
+
+import (
+	"github.com/supertokens/supertokens-golang/recipe/thirdparty/tpmodels"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
```

Comment:
this should only return:
- if tenant not in db -> static list
- if tenant in db -> return db list

The reason we do not want to return a union of static + db is cause user might define static AD and Okta, but in the db map one tenant to AD and another to Okta. If we give union, then both tenants will see both providers which won't work.

Reply:
Done